x11-packages/firefox/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://www.mozilla.org/firefox
 TERMUX_PKG_DESCRIPTION="Mozilla Firefox web browser"
 TERMUX_PKG_LICENSE="MPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="127.0.2"
-TERMUX_PKG_SRCURL=https://ftp.mozilla.org/pub/firefox/releases/${TERMUX_PKG_VERSION}/source/firefox-${TERMUX_PKG_VERSION}.source.tar.xz
```

Comment:
Changing `ftp.mozilla.org` to `archive.mozilla.org` as the latter supports IPv6 while the former dont. At least on from my local set up.
```
PS C:\Users\jylo> nslookup ftp.mozilla.org
Server:  dns.google
Address:  2001:4860:4860::8888

Non-authoritative answer:
Name:    ftp.mozilla.org
Address:  34.117.35.28

PS C:\Users\jylo> nslookup archive.mozilla.org
Server:  dns.google
Address:  2001:4860:4860::8888

Non-authoritative answer:
Name:    archive.mozilla.org
Addresses:  2600:1901:0:b9fd::
          34.117.35.28
```